.gitignore

```diff
@@ -66,6 +66,10 @@ package-lock.json
 # Local Netlify folder
 .netlify
 
+# OpenTitan test build
+boards/opentitan/earlgrey-nexysvideo/binary
+boards/opentitan/earlgrey-cw310/binary
```

Comment:
I feel like we are going a bit in a circle here. Should the root folder be `earlgrey` to match?

Reply:
No, I think this is correct as is. It's the general OpenTitan directory. It can then contain earlgrey or english breakfast boards (Tock only supports earlgrey at the moment).